main.tf

```diff
@@ -61,6 +61,7 @@ resource "ibm_is_vpn_server" "vpn" {
   client_authentication {
     method            = var.client_auth_methods
     identity_provider = var.client_auth_methods == "username" ? "iam" : null
+    client_ca_crn     = var.client_auth_methods == "certificate" ? var.server_cert_crn : null
```

Comment:
Although, for testing or non-prod environment we could use same certificate for server and client but realistically, I think we should use a separte variable here like `client_cert_crn`.

Reply:
good one, I exposed a new variable `client_cert_crn` which can contain multiple client certs.